pkg/util/inplaceupdate/inplace_update_defaults.go

```diff
@@ -339,6 +337,14 @@ func defaultCalculateInPlaceUpdateSpec(oldRevision, newRevision *apps.Controller
 		}
 		return nil
 	}
+	if utilfeature.DefaultFeatureGate.Enabled(features.InPlaceWorkloadVerticalScaling) &&
+		len(updateSpec.ContainerResources) != 0 {
+		// when container resources changes exist, we should check pod qos
+		if changed := verticalUpdateImpl.IsPodQoSChanged(oldTemp, newTemp); changed {
+			klog.InfoS("can not inplace update when qos changed")
```

Comment:
add pod namespace and name in the log?

Reply:
We can not get the information of pod in this function.
`defaultCalculateInPlaceUpdateSpec(oldRevision, newRevision *apps.ControllerRevision, opts *UpdateOptions) *UpdateSpec`